rewrite-core/src/main/java/org/openrewrite/config/DeclarativeRecipe.java

```diff
@@ -168,7 +168,7 @@ public boolean isAcceptable(SourceFile sourceFile, ExecutionContext ctx) {
 
     @EqualsAndHashCode(callSuper = false)
     @Value
-    static class BellwetherDecoratedRecipe extends Recipe {
+    static class BellwetherDecoratedRecipe extends Recipe implements DelegatingRecipe {
```

Comment:
Should `BellwetherDecoratedScanningRecipe` below implement the same interface?

Reply:
yes, good catch!